.chloggen/init_valkey_receiver.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: 'enhancement'
```

Comment:
this should be new_component

Reply:
Fixed https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/37005/commits/db230b1730f20bb70f4cbbb3626ae46cdb3b7888